### PR TITLE
`anni`: Layout

### LINKED ISSUE #346: `anni`: layout
- [x] fixed width app layout container
- [x] nav bar
- [x] completely ignore and neglect responsiveness for lack of time

### PR DESCRIPTION
**Pull request by jannis-baum**
Based on #330 

Closes #346

### CODE REVIEW

**Comment by Benjamin-Frost** [comment]
@jannis-baum can you rebase this branch? 😊

**Comment by jannis-baum** [comment]
> @jannis-baum can you rebase this branch? 😊

Done!☺️ Shouldn't there be some way of updating the diff Github shows? Most of this was already on main before the rebase, the diff just didn't refresh automatically

**Comment by Benjamin-Frost** [comment]
> > @jannis-baum can you rebase this branch? 😊
> 
> Done!☺️ @Benjamin-Frost Shouldn't there be some way of updating the diff Github shows? Most of this was already on main before the rebase, the diff just didn't refresh automatically.

I face this problem all the time. But unfortunately, I am not familiar with any way to solve this. You can only create links like [this](https://github.com/hpi-dhc/PharMe/pull/347/files/0800691f5e98acd4fbd0a4cd45ee499f171fe1c0..eee07567856abbdd846f1982898db4af421d29e3) to only show the new commits when reviewing. But they are kind of annoying and rather tricky to create when there are new pushes on the other branch.

File: annotation-interface/components/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+interface ITabDefinition {
```
Comment (Benjamin-Frost) [inline annotation-interface/components/NavBar.tsx:4]: In TypeScript, interfaces are conventionally **not** prefixed with an "I". See [this](https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#interface) for reference.

**Review by Benjamin-Frost — APPROVED** [review]
Only one minor thing. I will already approve because everything else lgtm!

File: annotation-interface/components/NavBar.tsx (hunk 1 above)
Comment (jannis-baum) [reply annotation-interface/components/NavBar.tsx]: Resolved in [ad0bb11](https://github.com/hpi-dhc/PharMe/pull/347/commits/ad0bb1179be193c35c45ca361f91c501f4ce87bc).

**Pull request merged**